### ISSUE TITLE
Publish for Scala 2.13.0-RC1

### ISSUE THREAD

**Issue by sadhen** [scalanlp/breeze#738]

**Comment by pomadchin** [comment]
In terms of this issue it would be also necessary to drop `Scala 2.10` to depend on a new spire (spire dropped Scala 2.10 since `0.16.x`).

**Comment by dlwh** [comment]
I've mostly got this working but I need spire to release a 2.13-RC1 build (skipping M5)

**Comment by pomadchin** [comment]
There is a problem with Spire 2.13-RC1 release actually https://github.com/typelevel/spire/issues/788

**Comment by dlwh** [comment]
Yeah I saw that. I'm going to wait on that

**Comment by hmf** [comment]
@dlwh Any way you can try with spire version `0.17.0-M1`? Listed in Maven as compatible with Scala 2.13. Should I open another issue or will this one do?

TIA

**Comment by dlwh** [comment]
I've been working on it. It's a huge pain to fix the custom collections and
I've been busy at work

**Comment by hmf** [comment]
@dlwh Thanks for the _heads up_. Hope I don't have to go through too much when moving to 2.13.

**Comment by dlwh** [comment]
got it compiling, will cut a release today or tomorrow.

**Comment by unoexperto** [comment]
@dlwh could you please publish artifacts for 2.13.0 ?

**Comment by dlwh** [comment]
done. version is 1.0-RC3

**Comment by unoexperto** [comment]
@dlwh Sorry, what repo is it in ? https://mvnrepository.com/artifact/org.scalanlp/breeze ?

**Comment by dlwh** [comment]
should be there now. use RC4
